frontend/src/e2e-playwright/specs/5_employee/vasu.spec.ts

```diff
@@ -86,70 +85,113 @@ describe('Vasu document page', () => {
   beforeEach(async () => {
     page = await Page.open()
     await employeeLogin(page, 'ADMIN')
-    vasuPage = await openDocument()
-  })
-
-  test('An unpublished vasu document has no followup questions', async () => {
-    await waitUntilEqual(() => vasuPage.followupQuestionCount, 0)
   })
 
-  describe('With a finalized document', () => {
-    const finalizeDocument = async () => {
-      await vasuPage.finalizeButton.click()
-      await vasuPage.modalOkButton.click()
-      await vasuPage.modalOkButton.click()
-      vasuPage = await openDocument()
-      await vasuPage.assertDocumentVisible()
-    }
-
-    beforeAll(async () => {
-      page = await Page.open()
-      await employeeLogin(page, 'ADMIN')
-      vasuPage = await openDocument()
-      await finalizeDocument()
-    })
-
-    test('A published vasu document has one followup question', async () => {
-      await waitUntilEqual(() => vasuPage.followupQuestionCount, 1)
-    })
-
-    test('Adding a followup comment renders it on the page', async () => {
-      vasuPage = await editDocument()
-      await vasuPage.inputFollowupComment('This is a followup')
-      await waitUntilEqual(
-        () => vasuPage.followupEntryTexts,
-        ['This is a followup']
+  describe('Fill out document', () => {
+    test('Fill the authors section', async () => {
+      const vasuEditPage = await editDocument()
+      const authors = vasuEditPage.authorsSection
+      await authors.primaryFirstNameInput.fill('Leena')
+      await authors.primaryLastNameInput.fill('Virtanen')
+      await authors.primaryTitleInput.fill('Johtaja')
+      await authors.primaryPhoneNumberInput.fill('1234565')
+
+      expect(await authors.otherFieldsCount).toBe(1)
```

Comment:
You cannot use `expect` in playwright tests. It fails if the page updates slowly. Use `waitUntilXXX` functions instead:

```
await waitUntilEqual(() => authors.otherFieldsCount, 1)
```